### DISCUSSION TITLE
FRSKY Twin LITE Pro 2.4Ghz Dual external Module

### DISCUSSION THREAD

**Discussion by sobrinof** [EdgeTX/edgetx#4834] [Ideas]
Hello All

FrSky released an external module that allows the following protocols: ACCESS, ACCST D16 and TW.
This module might be very useful for those users that prefer to use Edge TX in Radiomaster radios (for instance) because of the ACCESS and TW protocols.

Best Regards.

**Comment by gagarinlg** [comment]
This would require FrSky cooperating with us and us wanting to cooperate with FrSky.
Neither is probably happening.

**Reply by sobrinof** [reply]
Hello Again

I really do not know how it is done, but I have seen the Taranis SE 2019 using Edge TX and the internal ISRM Module (with Access). 
Probably it is made out of the official EdgeTX developing team.

Regards.

**Reply by gagarinlg** [reply]
This was done before EdgeTX and at a time where FrSky was embracing open source software.
FrSky has another focus now and is not willing to work with us.

**Comment by pfeerick** [comment]
And even then, it wasn't added to OpenTX, so while it would be good for end
users with compatible receivers, I doubt it will be supported.
